src/EditorFeatures/Core/Implementation/DocumentationComments/AbstractDocumentationCommentCommandHandler.cs

```diff
@@ -587,5 +569,143 @@ public void ExecuteCommand(InsertCommentCommandArgs args, Action nextHandler)
                     }
                 });
         }
+
+        public CommandState GetCommandState(OpenLineAboveCommandArgs args, Func<CommandState> nextHandler)
+        {
+            return nextHandler();
+        }
+
+        public void ExecuteCommand(OpenLineAboveCommandArgs args, Action nextHandler)
```

Comment:
Consider de-duping the implementation here with that of OpenLineBelow's ExecuteCommand below.